Source/WebCore/Modules/compression/DecompressionStreamDecoder.cpp

```diff
@@ -32,15 +32,121 @@
 
 namespace WebCore {
 
-RefPtr<Uint8Array> DecompressionStreamDecoder::decode(const BufferSource&& input)
+ExceptionOr<RefPtr<Uint8Array>> DecompressionStreamDecoder::decode(const BufferSource&& input)
 {
-    UNUSED_PARAM(input);
-    return nullptr;
+    size_t length = input.length();
```

Comment:
I'd just use `input.length()` later in the method, since this isn't used anywhere else.